src/components/Stepper/index.tsx

```diff
@@ -278,12 +288,10 @@ const Stepper: VoidFunctionComponent<StepperProps> = ({
             width: inputValue.toString().length * 10 + 15,
           }}
           value={inputValue.toString()} // A dom element can only have string attributes.
+          aria-label="Input"
         />
-
-        <StyledInput disabled={disabled} as="span">
-          {text}
-        </StyledInput>
-      </StyledCenterTouchable>
+        {text ? <StyledAbbr disabled={disabled}>{text}</StyledAbbr> : null}
```

Comment:
How can we be sure that it will be an abbreviation  ? Shouldn't we instead allow passing a node ?

Reply:
Not always, eg on Function :)

![Uploading Screenshot 2021-12-21 at 11.43.08.png…]()

Reply:
oh right I didn't see this one, I will change it then :)